recipe/build.sh

```diff
@@ -10,6 +10,10 @@ autoconf
 if [[ `uname` == 'Darwin' ]]; then
     # make check below fails on osx unless $PREFIX/lib is added to rpath
     LDFLAGS="$LDFLAGS -Wl,-rpath,${PREFIX}/lib"
+    if [[ "$CC" != "clang" ]]; then
+        # llvm-clang case
+        CFLAGS="$CFLAGS -I${PREFIX}/lib/clang/4.0.1/include"
```

Comment:
This is incorrect. Here's a minimal example of how to use `llvm-openmp` with the new compilers:

`echo -e "#include <omp.h>\nint main() {}" | CONDA_BUILD_SYSROOT=/opt/MacOSX10.9.sdk $CC -x c -fopenmp=libomp -`

You do need to add `llvm-openmp` as both a build and run requirement though. To fix that we could split `llvm-openmp` in two, one with a `-impl` suffix that only contains the library and headers and the other without that only contains a strong run export of `llvm-openmp-impl`.

Reply:
Thanks. I'm not 100% sure I understand the changes needed:
 
1.) Should I create a `conda_build_config.yml` file and then define `CONDA_BUILD_SYSROOT` there instead of within `build.sh`?  

2.)I had tried adding `-fopenmp=libomp` previously without success, but perhaps that is because I was missing the `-x c` when I tried it.  What is the final `-` for in the call to clang in the minimal example provided above?